rosdep/base.yaml

```diff
@@ -4328,6 +4332,7 @@ liborocos-kdl:
     '*': [liborocos-kdl1.4]
     bionic: [liborocos-kdl1.3]
     xenial: [liborocos-kdl1.3]
+    jammy: [liborocos-kdl1.5]
```

Comment:
I think CI is failing because this isn't alphabetically sorted?

Reply:
Done through change requested below :)